route53_crossfeed_app.tf

```diff
@@ -197,36 +197,65 @@ resource "aws_route53_record" "crossfeed_prod_ses_TXT" {
 # Staging entries
 # ------------------------------------------------------------------------------
 
-resource "aws_route53_record" "crossfeed_staging_A" {
+resource "aws_route53_record" "crossfeed_staging_cd_A" {
   provider = aws.route53resourcechange
 
   alias {
     name                   = "do8d5i2re25cf.cloudfront.net."
     zone_id                = var.cloudfront_zone_id
     evaluate_target_health = false
   }
-  name    = "staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "staging-cd.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
   type    = "A"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_AAAA" {
+resource "aws_route53_record" "crossfeed_staging_cd_AAAA" {
   provider = aws.route53resourcechange
 
   alias {
     name                   = "do8d5i2re25cf.cloudfront.net."
     zone_id                = var.cloudfront_zone_id
     evaluate_target_health = false
   }
-  name    = "staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "staging-cd.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "AAAA"
+  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+
+
+# The hosted_zone_id for the below records comes from https://docs.aws.amazon.com/general/latest/gr/elb.html
+# (NLBs in us-gov-west-1 region)
+resource "aws_route53_record" "crossfeed_staging_A" {
+  provider = aws.route53resourcechange
+
+  alias {
+    name                   = "crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com."
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+  name    = "crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "A"
+  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+
+resource "aws_route53_record" "crossfeed_staging_AAAA" {
+  provider = aws.route53resourcechange
+
+  alias {
+    name                   = "crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com."
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+  name    = "crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
Should this be something like `staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}`?  You already have an `AAAA` record with this name [here](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/78/files#diff-5191386bc2ec7103d66596882349fc3029e8e15deee5d010954e4f19ca32fe37R22-R33).

Reply:
Yeah, thanks for catching that -- updated.

Reply:
The `A` and `AAAA` records now apply cleanly.